.github/workflows/update-of-sdk.yml

```diff
@@ -0,0 +1,78 @@
+name: Update DevCycle SDKs to Latest
+on:
+  schedule:
+    - cron: "0 12 * * *"
+  workflow_dispatch:
+
+jobs:
+  update-dvc:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+      - uses: actions/setup-node@v3
+        with:
+          node-version: '20.x'
+          cache: 'yarn'
+
+      - name: Set Git author
+        shell: bash
+        run: |
+          git config --global user.email "github-tracker-bot@taplytics.com"
+          git config --global user.name "DevCycle Automation"
+
+      - name: Set branch name
+        shell: bash
+        run: echo "BRANCH_NAME=update-of-sdk" >> $GITHUB_ENV
+
+      - name: Update OpenFeature SDKs to latest
```

Comment:
Updating to latest isn't always what we want, can you make this a param to be either a specific version or latest?

Reply:
I haven't really seen a situation yet where we don't want to be on latest, going to keep this simple for now.